app/services/manifest_signer.rb

```diff
@@ -0,0 +1,46 @@
+class ManifestSigner
+  def initialize(args)
+    @args = args
+  end
+
+  def perform
+    cdx_start = Time.current
+
+    cdx_response = CDX::Manifest.new(parsed_args).sign
+
+    cdx_stop = Time.current
+    Rails.logger.debug(ANSI.blue{ "  CDX signature time: #{sprintf('%#g', (cdx_stop - cdx_start))} seconds" })
```

Comment:
I think it's good to have it somewhere, outside the CDX classes themselves. I'm interested in the total run time of the CDX action, from the perspective of the caller.